### PR TITLE
Switch to shaders-based Heatmap and remove old worker code

### PR DESCRIPTION
**Pull request by axelboc**
image

The shaders-based implementation is a lot lighter. We can remove three dependencies relating to web workers (61 kB off the unminified `main` bundle), and simplify the E2E tests a little since the renders should now be near instantaneous.

Looks like the shaders render the heatmaps the exact same way as before and that the E2E tests are more reliable (the delay is now down to 200):

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,4 +16,3 @@ npm-debug.log*
 *.log
 Thumbs.db
 .DS_Store
-.eslintcache
```
Comment (axelboc) [inline .gitignore:19]: In the latest patch version of CRA, this file has moved back to its proper location in the `node_modules/.cache` folder.

**Comment by axelboc** [comment]
Took me a lot longer than I wanted, but here is a new demo GIF for the `README` where the heatmap is super fast. I couldn't demo the slicing of the image stack because it was making the GIF too big.

demo

File: src/h5web/vis-packs/core/heatmap/Mesh.tsx [hunk 2]
```diff
@@ -1,61 +1,129 @@
...
-        <Html
-          className={styles.textureLoader}
-          data-visible={loading || undefined}
-        />
-      )}
-    </>
+    <mesh scale={[width, height, 1]}>
+      <planeGeometry attach="geometry" args={[1, 1]} />
+      <shaderMaterial attach="material" args={[shader]} />
+    </mesh>
```
Comment (loichuder) [inline src/h5web/vis-packs/core/heatmap/Mesh.tsx:125]: Why not keeping `width, height` on the geometry ?
```js
   <mesh>
      <planeGeometry attach="geometry" args={[width, height]} />
      <shaderMaterial attach="material" args={[shader]} />
    </mesh>
```

**Review by loichuder — APPROVED** [review]

File: src/h5web/vis-packs/core/heatmap/Mesh.tsx (hunk 2 above)
Comment (axelboc) [reply src/h5web/vis-packs/core/heatmap/Mesh.tsx:125]: If I do, the `uv` values are not between `0` and `1` but, I think, between `0` and `width` or `height`, which doesn't work well with the `texture()` function -- we get near-solid heatmap with only the "highest" colour in the colour map.

**Pull request merged**